frame/stake-tracker/src/lib.rs

```diff
@@ -0,0 +1,118 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright (C) 2023 Parity Technologies (UK) Ltd.
```

ditto for all of them.